### DISCUSSION TITLE
Latest Markdown changes causing problem

### DISCUSSION THREAD

**Discussion by HotCakeX** [community/community#68453] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

This code 

```html
## <img width="40" src="https://raw.githubusercontent.com/HotCakeX/Harden-Windows-Security/main/images/Gifs/surface.gif"> Which Device to Use ?
```

No longer renders the image. To fix it, I have to use this

```html
## <img width="40" src="https://raw.githubusercontent.com/HotCakeX/Harden-Windows-Security/main/images/Gifs/surface.gif"> [Which Device to Use ?](https://github.com/HotCakeX/Harden-Windows-Security/blob/main/Rationale.md#-which-device-to-use-)
```

The image is only rendered if the text next to it is a link.

This only applies to GIFs, works fine with PNGs.

You can see it here: https://github.com/HotCakeX/Harden-Windows-Security/blob/main/Rationale.md

P.S This problem started happening since few days ago.

**Comment by melvincwng** [comment]
Seems like an issue unfortunately 😕

Interestingly enough, the gif image is rendered when I tested in an instance of GitHub Codespaces.
